### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by binaruappuhamy**
*Note: 🔺Homepage modifications*

 - Summary 🔺 *Add a homepage banner with styling a functional project carousel*
 
 - Issue Number 🔺 *https://github.com/UofT-VEEP/VEEP-Website/issues/37*

- New Proposal *Figure out how to fit the banner height to the height on any screen. Add css styles for banner and banner element positioning for small/medium screens*

### CODE REVIEW

File: src/Components/HomePage/HeaderScreen.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-import { Container, Button } from 'react-bootstrap';
+import { Container, Button, Row, Col, Card } from 'react-bootstrap';
```
Comment (CatherineZM) [inline src/Components/HomePage/HeaderScreen.js:3]: The security check failed for `Card was never used`

File: src/Components/HomePage/Styles/FlipCard.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
+.CarouselCard{
+    height:740px;
```
Comment (CatherineZM) [inline src/Components/HomePage/Styles/FlipCard.css:2]: As we discussed in the meeting, explore the unit `vw` and `vh` for auto adjustment on size

File: src/Components/projects.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
+{
+  "tags_to_project": {
+    "homepage_carousel": [0, 1],
+    "internal_projects": [0],
+    "external_projects": [1] 
+  },
+  "proj_list": [
+    {
+      "project_name": "project 1",
```
Comment (CatherineZM) [inline src/Components/projects.json:9]: Take a look at `src/Components/ProjectsPage/Projects.js` and see if you can use those directly instead. Feel free to add a image component in there

File: src/HomePage.js [hunk 4]
```diff
@@ -21,10 +20,10 @@ class HomePage extends React.Component {
     return (
       <div>
         <NavBar />
-        <Container>
+        <div>
           <HeaderScreen />
-          <MainContent />
-        </Container>
+          <MainContent />
```
Comment (CatherineZM) [inline src/HomePage.js:25]: Remove the white space here

**Review by CatherineZM — CHANGES_REQUESTED** [review]
It looks good! Great job! Added some comments for minor changes

File: src/Components/HomePage/HeaderScreen.js (hunk 1 above)
Comment (binaruappuhamy) [reply src/Components/HomePage/HeaderScreen.js:3]: Fixed.

**Review by Felix-Deng — APPROVED** [review]
Thank you, Binaru! Looks good to me

**Comment by iiiizuo** [comment]
Thanks for the hard work. Looks great!

**Pull request merged**